recipe/meta.yaml

```diff
@@ -1,37 +1,43 @@
-{% set version = "3.1.1a1" %}
+{% set name = "pyopengl" %}
+{% set version = "3.1.7" %}
 
 package:
-  name: pyopengl
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: PyOpenGL-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/source/P/PyOpenGL/PyOpenGL-{{ version }}.tar.gz
-  sha256: c96d909b359abe3271b746bacf7e6ba52935141e2406a8f90231e4e44dfa4075
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyOpenGL-{{ version }}.tar.gz
+  sha256: eef31a3888e6984fd4d8e6c9961b184c9813ca82604d37fe3da80eb000a76c86
 
 build:
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   number: 0
+  skip: true # [win] pyopengl repacks opengl on windows.
```

Comment:
The wheel they ship on PyPI is a pure Python wheel and doesn't contain any DLL: https://inspector.pypi.io/project/pyopengl/3.1.7/packages/99/48/00e31747821d3fc56faddd00a4725454d1e694a8b67d715cf20f531506a5/PyOpenGL-3.1.7-py3-none-any.whl/ (`PyOpenGL-3.1.7-py3-none-any.whl`).

Maybe we could patch the `setup.py` to not bundle the DLLs? Though I'm not sure how this package works to be honest.